CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## `v10.4.0`
+
+This release upgrades our base image from Debian 10 to Debian 11.  
+There is also an important regression fixed for `SSL_TYPE=letsencrypt` users.
+
+- **[fix]** A regression with `check-for-changes.sh` introduced in `v10.3.0` affected `SSL_TYPE=letsencrypt`, preventing detection of cert renewals to restart services (_unless using `acme.json`_) [#2326](https://github.com/docker-mailserver/docker-mailserver/pull/2326)
+- **[improvement]** Base image upgraded from Debian 10 Buster to Debian 11 Bullseye [#2116](https://github.com/docker-mailserver/docker-mailserver/pull/2116)
+  - Postfix upgraded from `3.4` to `3.5`. Dovecot upgraded from `2.3.4` to `2.3.13`. Python 2 is no longer included in the image, Python 3 remains (_[more information](https://github.com/docker-mailserver/docker-mailserver/pull/2116#issuecomment-955615529)_).
+  - **WARNING:** `fail2ban` has a CVE that may affect users who configure it to send email (_[more information](https://github.com/docker-mailserver/docker-mailserver/pull/2116#issuecomment-955800544)_).
```

Comment:
Afaik, only the debian package of F2B is affected by this. We are using the official '0.11.2' release, which was released before the mentioned patch. There was no new official release since then.